docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md

```diff
@@ -80,6 +90,7 @@ An instance in cloud computing is a server resource provided by third-party clou
 Cloud providers offer a variety of instance types across the regions where they have data centers. By default, where possible, YugabyteDB Managed uses the following instance type families for dedicated clusters:
```

Comment:
Don't think we need the whole Instance Types section. What instance types we use is internal Yugabyte details.